backend/Squirrel.ConsoleApp/appsettings.json

```diff
@@ -1,12 +1,15 @@
 ﻿{
-    "Logging": {
-        "LogLevel": {
-            "Default": "Debug",
-            "System": "Information",
-            "Microsoft": "Information"
-        }
-    },
-    "WebServerSettings": {
-        "BaseUrl": "http://localhost:44567"
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information"
     }
+  },
+  "WebServerSettings": {
+    "BaseUrl": "http://localhost:44567"
+  },
+  "SignalRSettings": {
+    "ConnectionUrl": "http://localhost:5050/SquirrelHub"
```

Comment:
maybe use more descriptive name ("HubConnectionUrl" or something like that) instead of "ConnectionUrl", it's important to show that it is a hub